erts/etc/common/escript.c

```diff
@@ -106,35 +106,36 @@ get_env(char *key)
 {
 #ifdef __WIN32__
     DWORD size = 32;
-    char  *value=NULL;
-    wchar_t *wcvalue = NULL;
     wchar_t wckey[256];
-    int len; 
+    int len;
 
     MultiByteToWideChar(CP_UTF8, 0, key, -1, wckey, 256);
     
-    while (1) {
-	DWORD nsz;
-	if (wcvalue)
```

Comment:
When `wcvalue` was not returned or breaked, there will generate remain memory with size of wchar_t.

Reply:
Are you sure? What environment value would trigger it? From what I can tell the loop will only exit at the two returns, which both free wcvalue before leaving.